conf/production.yaml

```diff
@@ -492,8 +492,7 @@ import:
         url: 'https://yt-dl.org/downloads/latest/youtube-dl'
```

Comment:
The URL need to be changed for yt-dlp:

https://api.github.com/repos/yt-dlp/yt-dlp/releases